src/compiler/CombineJavascriptCompiler.ts

```diff
@@ -204,11 +211,7 @@ export class CombineJavascriptCompiler implements ICompiler {
                         this.assetsPath,
                         '.' + includeLine
                     );
-                    const relativePath = path.relative(
-                        this.assetsPath,
-                        resolvedFile
-                    );
-                    result.push(`.${path.sep}${relativePath}`);
+                    result.push(resolvedFile);
```

Comment:
@slaven3kopic using relative paths didn't work for me anymore. Maybe it's because of webpack or maybe it's only on windows. Can you try if it still work for you like this?

Reply:
@vladimir-arsov The solution you proposed works for me. Compilation was successful and everything seems fine after running cplace